BackofficeBundle/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -190,3 +195,11 @@
         {% endif %}
     {% endspaceless %}
 {% endblock oo_date_picker_widget %}
+{% block form_legend %}
+    <div class="help-block">
+        <p>* {{ 'open_orchestra_backoffice.form.orchestra_fields.required_field' | trans }}</p>
+        {% if template_selection_helper is defined and template_selection_helper == true %}
```

Comment:
I would put a variable name which is more generic, this variable is only linked to the template selection and the ** is for much more